lib/exercises.rb

```diff
@@ -1,19 +1,66 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: n*mlog(m) where m is the size of the words and n is the number of words. nm2 (nm squared) is the worst case. (I am confused here)
+# Space Complexity: O(n) for the making of the hash and return value
 
 def grouped_anagrams(strings)
```

Comment:
👍 
If you can assume the words are limited in size the time complexity is O(n), otherwise you're right on time complexity.